app/assets/stylesheets/pages/forgot_password_page.scss

```diff
@@ -0,0 +1,11 @@
+.forgot-password-container {
+  min-height: 100vh;
+}
+
+.forgot-password-form {
+  width: 250px;
```

Comment:
maybe set this as the `min-width` and expand as the screen gets bigger (it would also be good to set a `max-width` like `500px` or so....but i think 250 is probably fine. can you put pics in your future PRs?